cartridges/int_turnto_core_v5/cartridge/scripts/jobsteps/ExportCatalog.js

```diff
@@ -287,28 +294,27 @@ function process(product) {
             localesArray.forEach(function (l) {
                 request.setLocale(l);
                 var url = URLUtils.http('Product-Show', 'pid', product.getID()).toString();
-                var item = {
+                localeData[l] = {
                     title: TurnToHelper.sanitizeStr(product.getName(), ' '),
                     itemUrl: url,
                     mobileItemUrl: url
                 };
-                localeData[l] = item;
             });
 
             var defaultLocale = Site.getCurrent().getDefaultLocale();
             request.setLocale(defaultLocale);
             // build locale JSON
-            var localejson = {
+            json[locales] = {
                 sku: TurnToHelper.replaceNull(product.getID(), ''),
                 imageurl: imageURL,
                 title: TurnToHelper.sanitizeStr(product.getName(), ' '),
                 price: TurnToHelper.replaceNull(priceStr, ''),
                 currency: TurnToHelper.replaceNull(price.getCurrencyCode(), ''),
-                active: product.getAvailabilityModel().isOrderable() ? 'Y' : 'N',
+                active: product.isOnline() ? 'Y' : 'N',
                 itemurl: URLUtils.http('Product-Show', 'pid', product.getID()).toString(),
                 category: '', // Leaving blank because CATEGORYPATHJSON is populated
                 keywords: TurnToHelper.sanitizeStr(keywords, ' '),
-                instock: product.getOnlineFlag() ? 'Y' : 'N',
+                instock: product.getAvailabilityModel().isOrderable() ? 'Y' : 'N',
```

Comment:
isOrderable: Returns true if the Product is currently online (based on its online flag and online dates) and is orderable in its minimum-order-quantity. If the product does not have a minimum-order-quantity specified, then 1 is used. The method returns false otherwise.
Note: Orderable status is more general than in-stock status. A product may be out-of-stock but orderable because it is back-orderable or pre-orderable.

Reply:
makes sense